.circleci/config.yml

```diff
@@ -161,18 +161,15 @@ jobs:
       - restore-cache:
           keys:
             - cabal-{{ .Environment.HIE_CACHE }}-{{ arch }}-{{ .Environment.CIRCLE_JOB }}
-      - run:
-          name: Cabal version
-          command: cabal --version
       - run:
           name: Update
-          command: cabal new-update
+          command: cabal update
       - run:
           name: Configure
-          command: cabal new-configure --enable-tests
+          command: cabal configure --enable-tests
       - run:
           name: Build
-          command: cabal new-build -j1 # need j1, else ghc-lib-parser triggers OOM
+          command: cabal build -j1 # need j1, else ghc-lib-parser triggers OOM
```

Comment:
I think it would be better if this was v2-* for forward and backwards compatability. Also, the version print is useful to make sure you are executing with the right version, why did you remove it?

Reply:
In my mind, the cabal job is not about backwards compatibility but rather ensuring that the project builds with the vanilla cabal setup. i.e. if the default build behaviour changes we want this job to break so we can catch it.

As for the version command I just assumed it snuck in whilst debugging (d1978e6dce5384b83888de257b0b310be2258418) so I just removed it to tidying things up a bit. The version of cabal-install can be checked from the docker image, but it should be the one that ships with the ghc distribution anyway